rewrite IPsec remote access documentation (#72)
opnsense/docs: source/manual/how-tos/ipsec-rw-srv-rsamschapv2.rst

=====================================================
IPsec: Setup OPNsense for IKEv2 Mutual RSA + MSCHAPv2
=====================================================

.. contents:: Index

Mutual RSA + MSCHAPv2 via IKEv2 is based on client certificate authentication combined with username
and password via MSCHAPv2.

> Be sure to install the client certificate on your enduser device.

Reviewer: ```suggestion
Be sure that the client certificate is installed on your users device.
```

Phase 1 proposal (Authentication)
---------------------------------
=========================== ======================= ============================================
 **Authentication method**   Mutual RSA + MSCHAPv2  *This is the method we want here*
 **My identifier**           Distinguished Name     *Set the FQDN you used within certificate*
 **My Certificate**          Certificate            *Choose the certificate from dropdown list*
=========================== ======================= ============================================

Phase 1 proposal (Algorithms)
-----------------------------
========================== ============= ===========================================================
 **Encryption algorithm**   AES           *For our example we will use AES/256 bits*
 **Hash algoritm**          SHA1,SHA256   *SHA1 and SHA256 for compatibility*
 **DH key group**           1024,2048 bit *1024 and 2048 bit for compatibility*
 **Lifetime**               28800 sec     *lifetime before renegotiation*
========================== ============= ===========================================================

Advanced Options are fine by default.

> **Save** your setting.

Reviewer: ```suggestion
**Save** your settings.
```
